great_expectations/cli/toolkit.py

```diff
@@ -348,3 +368,9 @@ def load_data_context_with_error_handling(directory: str) -> DataContext:
     except ge_exceptions.PluginClassNotFoundError as err:
         cli_message(err.cli_colored_message)
         sys.exit(1)
+
+
+def confirm_proceed_or_exit():
+    if not click.confirm("OK to proceed?", default=True):
+        cli_message("Okay, bye!")
+        exit(0)
```

Comment:
If the message were parameterized you could replace 2-3 other instances of custom code in the init function with this!

Reply:
@Aylr This is meant a method that eventually all CLI commands will use to create consistency. I added a comment to the method.